hootl/lasagna/PastaSim.cpp

```diff
@@ -30,15 +28,16 @@ void PastaSim::init(){
 
 float PastaSim::evolve(float  action){
 	time += 1000/conf.freq;
-	if(time % (conf.sun_calc_interval*1000) == 0){
+	if(time/(conf.sun_calc_interval*1000) >= sun_pred_ctr){
```

Comment:
I'm not a huge fan of the fact that time and sun_calc_interval are in different units. If changing this is too hard though, renaming it to `sun_calc_interval_s` could help a lot, as could adding a little comment after the `time += 1000/conf.freq;` saying its units.